### ISSUE TITLE
Not mapping nested _id fields to id

### ISSUE THREAD

**Issue by MightyPixel** [balderdashy/sails-mongo#207]
Hi,

I'm getting an object that has an embedded object in it. Example:
```json
{
  _id: "...",
  name: "Big Object",
  brother: { _id: ".....", name: "Embedded Object" }
}
````
When I return the Big Object to some rest client I get the json:
```json
{
 id: "...",
 name: "Big Object",
 brother: { _id: ".....", name: "Embedded Object" }
}
```
Why the Embedded Object's has the key _id? Shouldn't it be named id as in the Big Object?

Cheers!

Versions:
-  "sails-mongo": "0.10.4"
-  "sails": "0.10.5"
-  mongo 2.4.8

**Comment by MightyPixel** [comment]
Anyone?

**Comment by sailsbot** [comment]
Thanks for posting, @MightyPixel.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
